client/packages/components/src/components/explorer/index.tsx

```diff
@@ -120,12 +120,17 @@ export const Explorer = (_props: WithOptional<ExplorerProps>) => {
   const [_explorerState, _setExplorerState] = useState<ExplorerNav | null>(
     null,
   );
+
   const props: WithDefaults<ExplorerProps> = fillPropsWithDefaults(
     _props,
     _explorerState,
     _setExplorerState,
   );
 
+  if (!props.adminToken || !props.appId) {
+    throw new Error('Admin token and app ID are required');
```

Comment:
small nit: imo we should have specific errors for each one. i.e if not appId. if not adminToken